testing/multi-env-test-engine/src/main/java/org/projectnessie/junit/engine/MultiEnvTestFilter.java

```diff
@@ -46,39 +44,27 @@ private Optional<Class<?>> classFor(TestDescriptor object) {
     return Optional.empty();
   }
 
-  private FilterResult filter(Class<?> testClass, UniqueId id) {
-    // Use the static extension data collected during the discovery phase.
-    // It is possible to reload extensions based of class objects from test descriptors,
-    // however that would add unnecessary overhead.
-    MultiEnvExtensionRegistry registry = registry();
-
+  /**
+   * This filter effectively routes all tests via the {@link MultiEnvTestEngine}, both actual
```

Comment:
In that case could we simply disable the default "jupiter" engine somehow?

Reply:
No